templates/todo/projects/csharp-cosmos-sql-func/infra/resources.bicep

```diff
@@ -0,0 +1,307 @@
+param location string
+param principalId string = ''
+param resourceToken string
+param tags object
+
+resource web 'Microsoft.Web/staticSites@2021-03-01' = {
+  name: 'stapp-${resourceToken}'
```

Comment:
Jon has extracted prefixes for common resource types into a separate, shared file. He is also working on making our Bicep docs more modular; you probably want to coordinate with him.

Reply:
Yes, please check main for how we are doing abbreviations now. https://github.com/Azure/azure-dev/blob/main/templates/todo/projects/csharp-cosmos-sql/infra/resources.bicep

Reply:
Resolved in latest commit.